### PR TITLE
vmui: correct the description of shortcut keys

### PR DESCRIPTION
**Pull request by Loori-R**
- correct the description of shortcut keys
- add a hotkey to call up the modal window for shortcut keys (`Cmd` + `/` for macOS or `F1` for other platforms.)
- refactor the component for the list of shortcut keys.

### CODE REVIEW

File: app/vmui/packages/vmui/src/components/Chart/GraphTips/contants/tips.tsx [hunk 1]
```diff
@@ -51,15 +51,16 @@ const legendTips = [
     </>
   },
   {
-    title: "Copy key-value pairs",
+    title: "Copy label key-value pairs",
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/components/Chart/GraphTips/contants/tips.tsx:54]: nit: label it is a key label pair

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: app/vmui/packages/vmui/src/components/Chart/GraphTips/contants/tips.tsx (hunk 1 above)
Comment (valyala) [reply app/vmui/packages/vmui/src/components/Chart/GraphTips/contants/tips.tsx:54]: I think it is OK to say `label key-value pair`

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request!

**Pull request merged**